src/triggers/messages.ts

```diff
@@ -0,0 +1,42 @@
+
+import { Table } from '../db/db';
+import { CrdtMessage, CrdtOpetation } from '../types/message';
+import { Cursor, Trigger } from '../types/trigger';
+
+export const pendingMempoolMessages: (tables: string) => Trigger<undefined> = 
+  (table) => {
+    return async (clients) => {
+      const sql = `
```

Comment:
looks good. given this is fairly complex/hard to read, probably should put a comment here describing the intention behind this query and explaining the expected functionality more clearly